releasenotes/notes/add_series_list_view-bf219022216fea6a.yaml

```diff
@@ -0,0 +1,11 @@
+---
+prelude: >
+    Some projects can have hundreds of patches actives at once, by giving the user
+    the ability to have an overview of all of the series makes knowing what's the
+    current state of a project much more simple.
+features:
+  - |
+    Add series-list view, a view containing all of the series for a project and that also
+    implements the SeriesBulkUpdatePatchesForm, allowing the user to edit all of the patches
+    within a series using the web ui.
+    Add series-detail view, a view containing details about a series and all of it's patches
```

Comment:
Can you be a little more generic here since `series-list` and `series-detail` mean nothing to a user of Patchwork that hasn't read the code. How about:

```
A series view is now available, allowing users to list available series and
view details of individual series.
```